<UI version="4.0" stdsetdef="1" >
 <class>TopicChooser</class> 
 <widget class="QDialog" name="TopicChooser" >
  <property name="objectName" >
   <string notr="true">TopicChooser</string>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Choose Topic</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>  
  <layout class="QVBoxLayout" >
   <property name="objectName" >
    <string notr="true">unnamed</string>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="objectName" >
      <string notr="true">label</string>
     </property>
     <property name="text" >
      <string>&amp;Topics</string>
     </property>
     <property name="buddy" stdset="0" >
      <cstring>listWidget</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget" >
     <property name="objectName" >
      <string notr="true">listWidget</string>
     </property>     
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Layout16" >
     <property name="objectName" >
      <string notr="true">Layout16</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="objectName" >
       <string notr="true">unnamed</string>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer name="Horizontal Spacing2" >
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="sizeType" >
         <enum>Expanding</enum>
        </property>
        <property name="orientation" >
         <enum>Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDisplay" >
        <property name="objectName" >
         <string notr="true">buttonDisplay</string>
        </property>
        <property name="text" >
         <string>&amp;Display</string>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>        
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel" >
        <property name="objectName" >
         <string notr="true">buttonCancel</string>
        </property>
        <property name="text" >
         <string>&amp;Close</string>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>        
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
</UI>
